cg/services/file_delivery/file_formatter_service/delivery_file_formatting_service.py

```diff
@@ -0,0 +1,16 @@
+from abc import abstractmethod, ABC
+
+from cg.services.file_delivery.fetch_file_service.models import DeliveryFiles
+from cg.services.file_delivery.file_formatter_service.models import FormattedFiles
+from cg.store.store import Store
+
+
+class DeliveryFileFormattingService(ABC):
```

Comment:
The other services are called `<Action>DeliveryFileService`, for example `MoveDeliveryFilesService` or `FetchDeliveryFilesService`. Wouldn't it be better to standardise this, and therefore rename this class to `FormatDeliveryFileService`?